### PR TITLE
Add "Darker" Theme + Padding Edits

### PR DESCRIPTION
**Pull request by phoebe-leong**
I added padding to the server navigation bar as well as the section headers and darkened the dark theme.

### CODE REVIEW

File: webpage/style.css [hunk 1]
```diff
@@ -530,14 +535,18 @@ textarea {
...
 	margin-top: .1in;
 	margin-bottom: .1in;
 	padding-left: 5px;
 	text-overflow: ellipsis;
 	overflow-x: hidden;
 	white-space: nowrap;
-	max-width: 220px;
+}
+
+#serverName {
```
Comment (DEVTomatoCake) [inline webpage/style.css:547]: not going to work unless you change it to the following, as there's no element with this ID:
```suggestion
#servername {
```

**Comment by kalebplayz125** [comment]
i quite like the darker theme lol

**Comment by phoebe-leong** [comment]
Yeah sure, I can make it to be "Darker Theme" lol

**Comment by MathMan05** [comment]
this merge is broken

**Comment by phoebe-leong** [comment]
yeah i know
im gonna make the changes :P

**Pull request closed without merging**